tools/Status/Status.py

```diff
@@ -506,9 +555,39 @@ def render_status(
         "See `spectre -h` for its path."
     ),
 )
-def status_command(refresh_rate, **kwargs):
+@click.option(
+    "--columns",
+    "-c",
+    "input_columns",
+    cls=PythonLiteralOption,
+    default=_default_fields_for_print(),
+    show_default=True,
+    help=(
+        "List of columns that will be printed for all jobs (executable"
+        " specific columns will be added in addition to this list). This can"
+        " also be specified in the config file with the name 'input_columns'."
+        " Note that if the '--allusers'  option is not specified, then the"
+        ' "User" column will be omitted. Specify the columns as a list like'
+        " so:"
+        "\b\n\n"
+        '-c \'["Col1", "Col2", "Col3"]\''
+        "\b\n\n"
+        " "
+    ),
+)
+@click.option(
+    "--available-columns",
+    "-e",
+    is_flag=True,
+    help="Print a list of all available columns to use.",
+)
+def status_command(refresh_rate, available_columns, **kwargs):
     """Gives an overview of simulations running on this machine."""
 
+    if available_columns:
+        rich.print(f"Available columns: {list(_available_fields().keys())}")
```

Comment:
```suggestion
        rich.print(rich.columns.Columns(_available_fields().keys()))
```

No need to print "Available columns: ". The user knows what's going on because they explicitly specified `--available-columns`.